2_isolation/README.md

```diff
@@ -8,17 +8,24 @@ In addition we were asked to develop our own heuristics to attempt to beat the s
 
 ## Instructions
 
-Play against AI:
+* Switch to Python 3.6 or greater environment
+
+* Install necessary dependencies:
+```bash
+$ pip install termcolor
```

Comment:
This is handled by pipenv install in the main README.md instructions. Pipenv replaces both virtualenv and pip. Its awesome and the future: https://github.com/kennethreitz/pipenv

Also the python 3.6 requirement is already specified in the main README.md and the tests with Travis CI are set to python 3.6.